readme/install.md

```diff
@@ -92,7 +92,7 @@ The executable will be installed to `/usr/bin`.
 
 ## macOS
 
-### With [Homebrew](https://brew.sh/)
+### With [HomeBrew](https://brew.sh/)
```

Comment:
I don't believe that it's HomeBrew, everywhere on https://brew.sh/ is Homebrew.

Reply:
You are right. I will fix tomorrow. I'm not in front of a computer now.